### ISSUE TITLE
Start fails

### ISSUE THREAD

**Issue by horodnicdragos** [VulcanJS/Vulcan-Starter#38]
Greetings!

I believe something went wrong with the 1.8.10 update and I cannot start vulcan from the two repo method. The setps to reproduce are fairly easy, just clone the two repos, install all required node packages and run like below:

METEOR_PACKAGE_DIRS="~/Documents/Vulcan/packages" meteor --port 3000 --settings settings.json
[[[[[ ~/Documents/market/Vulcan-Starter ]]]]] 

=> Started proxy.                             
=> Errors prevented startup:                  
   
   While selecting package versions:
   error: No version of vulcan:core satisfies all constraints: @1.8.10
   Constraints on package "vulcan:core":
   * vulcan:core@1.8.10 <- top level
   * vulcan:core@1.8.3 <- vulcan:i18n-en-us 1.8.3
   * vulcan:core@1.8.5 <- getting-started 0.0.0
   * vulcan:core@1.3.1 <- vulcan:forms 1.3.1 <- example-simple 0.0.0
   * vulcan:core@1.8.6 <- vulcan:accounts 1.8.6 <- example-simple 0.0.0
   * vulcan:core@1.8.10 <- example-simple 0.0.0
   
Meteor version 1.6.0.1
Node 9.7.1

I have an older clone which works in the two repo fashion, but i don't know if it's helping by much, it might be behind many commits. Apparently the same thing happens when running with npm start as well.

**Comment by gabimoncha** [comment]
Same here on Meteor 1.6.1 and Node 8.9.1

**Comment by SachaG** [comment]
Thanks, I'll take a look today

**Comment by SachaG** [comment]
I'm not seeing anything wrong locally, so can you try just deleting your `.meteor/versions` file in your copy of the starter repo to force a package refresh, and/or making sure all versions of the `vulcan:*` packages in there are set to `1.8.10`?

**Comment by SachaG** [comment]
Actually there were a couple packages that still had old versions. Can you pull and try again?

**Comment by gabimoncha** [comment]
@SachaG there is no versions file in `.meteor/`. Also, all the packages are set to 1.8.10
I pulled the repository again and I have the same issue.

**Comment by severnsc** [comment]
I'm getting this same error using the regular install. Installed Meteor, cloned repo, ran `npm install` and then `npm start` and get the error:

```
While selecting package versions:
   `error: No version of vulcan:core satisfies all constraints: @1.8.11`
   Constraints on package "vulcan:core":
   * vulcan:core@1.8.11 <- top level
   * vulcan:core@1.8.3 <- vulcan:i18n-en-us 1.8.3
   * vulcan:core@1.8.11 <- getting-started 0.0.0
   * vulcan:core@1.8.9 <- vulcan:forms 1.8.9 <- getting-started 0.0.0
   * vulcan:core@1.8.5 <- vulcan:accounts 1.8.5 <- getting-started 0.0.0
```

**Comment by SachaG** [comment]
Oh ok my bad, I thought I had published v1.8.11 of the packages but I hadn't :( It should be good now, can you try again? Really sorry about all this…

**Comment by severnsc** [comment]
It's working for me now! Thanks.
